package.json

```diff
@@ -115,6 +118,41 @@
         "./packages/template-kit:generate:docs",
         "./packages/node-nursery:generate:docs"
       ]
+    },
+    "check": {
+      "dependencies": [
+        "syncpack:check",
+        "check:format",
+        "lint",
+        "test"
+      ]
+    },
+    "syncpack:check": {
+      "command": "syncpack list-mismatches --config syncpack.config.ts --types all",
+      "files": [
+        "package.json",
+        "package-lock.json",
+        "packages/*/package.json",
+        "packages/*/package-lock.json"
+      ]
+    },
+    "syncpack:check:prod": {
+      "command": "syncpack list-mismatches --config syncpack.config.ts --types prod",
+      "files": [
+        "package.json",
+        "package-lock.json",
+        "packages/*/package.json",
+        "packages/*/package-lock.json"
+      ]
+    },
+    "syncpack:check:dev": {
+      "command": "syncpack list-mismatches --config syncpack.config.ts --types dev",
+      "files": [
+        "package.json",
+        "package-lock.json",
+        "packages/*/package.json",
+        "packages/*/package-lock.json"
+      ]
```

Comment:
Small wireit suggestions:

- You can make a "files-only" script when you want to refer to the same set of input files in multiple places.

- Adding `"files": [], "output": []` allows these scripts to be cached (because wireit doesn't cache unless you reassure it about all of the possible inputs/outputs).

```suggestion
    "syncpack:files": {
      "files": [
        "syncpack.config.ts",
        "package.json",
        "package-lock.json",
        "packages/*/package.json",
        "packages/*/package-lock.json"
      ]
    },
    "syncpack:check": {
      "command": "syncpack list-mismatches --config syncpack.config.ts --types all",
      "dependencies": [
        "syncpack:files"
      ],
      "files": [],
      "output": []
    },
    "syncpack:check:prod": {
      "command": "syncpack list-mismatches --config syncpack.config.ts --types prod",
      "dependencies": [
        "syncpack:files"
      ],
      "files": [],
      "output": []
    },
    "syncpack:check:dev": {
      "command": "syncpack list-mismatches --config syncpack.config.ts --types dev",
      "dependencies": [
        "syncpack:files"
      ],
      "files": [],
      "output": []
```

Reply:
Ooooooh nice!